### PR TITLE
Update the issue forms

### PR DESCRIPTION
**Pull request by Pandapip1**
This will allow us to point the issue toward the correct repository.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: pull-request
+    attributes:
+      label: Pull Request
+     description: Link to the pull request where the issue occured
```
Comment (MicahZoltu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:15]:
```suggestion
occurred
```

**Comment by MicahZoltu** [comment]
I feel like we shouldn't be receiving bot bug reports on this repository, they should be in the appropriate bot repository.  What is the argument for tracking bot bugs here?

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: pull-request
+    attributes:
+      label: Pull Request
+occurred
```
Comment (Pandapip1) [inline .github/ISSUE_TEMPLATE/bug-report.yml:15]: @MicahZoltu I think you accidentally committed to this PR

**Comment by Pandapip1** [comment]
Not sure why this got closed.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: pull-request
+    attributes:
+      label: Pull Request
+      description: Link to the pull request where the issue occured
```
Comment (MicahZoltu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:15]:
```suggestion
      description: Link to the pull request where the issue occurred
```

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (MicahZoltu) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: It was on purpose, I was trying to fix the spelling error that codespell gave.  I have no idea how I failed so badly here.  😖  Tried again, I think I got it right this time.  😬

**Comment by MicahZoltu** [comment]
So is the idea that users would file a bug here, and then an editor would come around and migrate it to the appropriate repository and then close the issue here?

I'm generally a little concerned about anything that increase editor workload, though you and Sam have been pretty actively lately so maybe we have a little breathing room for additional minor work... assuming you both stick around.  😬

**Comment by Pandapip1** [comment]
> I'm generally a little concerned about anything that increase editor workload, though you and Sam have been pretty actively lately so maybe we have a little breathing room for additional minor work... assuming you both stick around. 😬

I think with how little bugs there have been (that haven't *been* reported by me or Sam), I think the increase in editor workload is less than the increase in editor workload to update those paragraphs.

**Review by MicahZoltu — APPROVED** [review]

**Pull request merged**